arangod/CMakeLists.txt

```diff
@@ -218,6 +218,7 @@ set(LIB_ARANGO_PREGEL_SOURCES
 )
 
 set(LIB_ARANGO_AQL_SOURCES
+  ../3rdParty/date/src/tz.cpp
```

Comment:
hm, I doubt that adding `tz.cpp` twice will work?

Reply:
Now its exactly like libnoise